src/audio-worklet/free-queue/example/build/main.js

```diff
@@ -0,0 +1,3126 @@
+
```

Comment:
Is this the generated file? If so, please change the file name to `main.wasm.js`.

Reply:
After looking into it, I am not sure why this file is needed. Please add explanation to the commit description.

Reply:
This is the output file that emscripten generates for loading WebAssembly Module into worker.
Should We upload generated outputs with code or just .gitignore then and depend on user to build them for local testing?